assets/css/general.css

```diff
@@ -26,6 +26,116 @@ h2 {
     z-index: 10000;
 }
 
+/* sass-lint:disable class-name-format */
+.position-relative {
+    position: relative;
+}
+
+.d-none {
+    display: none;
+}
+
+.d-inline-block {
+    display: inline-block;
+}
+
+.w-0 {
+    width: 0%;
+}
+
+.w-100 {
+    width: 100%;
+}
+
+.border-0 {
+    border: 0;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.position-absolute {
+    position: absolute;
+}
+
+.font-weight-bold {
+    font-weight: bold;
+}
+/* sass-lint:enable class-name-format */
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#gsocRef,
+#codeInRef {
+    text-decoration: none;
+}
+
+#gsocLogo {
+    height: 2.5em;
+}
+
+#codeInLogo {
+    height: 2.5em;
+    padding-left: .5em;
+}
+
+#bytemarkLogo {
+    margin-top: .5em;
+}
+
+#sourceForgeLogo {
+    margin-top: .25em;
+}
+
+#creativeCommonsLicence,
+#gnuGplLicence {
+    margin-bottom: 0;
+    margin-left: .5em;
+    margin-right: .5em;
+    margin-top: 0;
+}
+
+#infoModalApertiumLogo {
+    height: 5em;
+    max-width: 100%;
+}
+
+#morphAnalyzerOutput {
```

Comment:
As I said on the previous PR, this belongs in https://github.com/apertium/apertium-html-tools/blob/master/assets/css/analysis.css. Please do not open new PRs each time.

Reply:
Sorry @sushain97 i am new to git so i will try pushing to this branch and update this branch always and also regarding file upload and progress bar i will build and check and let you know if something goes wrong.I am bit slow as my exams are going on so sorry for delay.